### PR TITLE
Features/search places page mockup

### PR DESCRIPTION
**Pull request by aLLeXUs**

### CODE REVIEW

File: frontend/config/dev.env.js [hunk 1]
```diff
@@ -3,5 +3,7 @@ const merge = require('webpack-merge')
 const prodEnv = require('./prod.env')
 
 module.exports = merge(prodEnv, {
-  NODE_ENV: '"development"'
+  NODE_ENV: '"development"',
+  MAPBOX_TOKEN: '"'+process.env.MAPBOX_TOKEN+'"',
```
Comment (nemoipaha) [inline frontend/config/dev.env.js]: Why we have such '"'+process.env.MAPBOX_TOKEN+'"' here?

File: frontend/config/prod.env.js [hunk 2]
```diff
@@ -1,4 +1,6 @@
 'use strict'
 module.exports = {
-  NODE_ENV: '"production"'
+  NODE_ENV: '"production"',
+  MAPBOX_TOKEN: '"'+process.env.MAPBOX_TOKEN+'"',
```
Comment (nemoipaha) [inline frontend/config/prod.env.js]: Same as  above

File: frontend/src/App.vue [hunk 3]
```diff
@@ -20,6 +20,7 @@ export default {
 
 <style>
 @import url(buefy/lib/buefy.css);
+@import url('https://api.tiles.mapbox.com/mapbox-gl-js/v0.47.0/mapbox-gl.css');
```
Comment (nemoipaha) [inline frontend/src/App.vue]: No need for cdn here. Mapbox has .css file in package

File: frontend/src/components/PlacesList/PlaceListComponent.vue [hunk 4]
```diff
@@ -89,6 +89,12 @@
 <style lang="scss" scoped>
```
Comment (nemoipaha) [inline frontend/src/components/PlacesList/PlaceListComponent.vue]: This file permissions changed 644-755. Please remove

File: frontend/src/components/place/SearchPlace.vue [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+
+    export default {
+        name: "SearchPlace",
+        components: {
+            PlaceListComponent,
+            Mapbox,
+        },
+        data() {
+            return {
+                mapBoxToken: process.env.MAPBOX_TOKEN,
```
Comment (nemoipaha) [inline frontend/src/components/place/SearchPlace.vue]: Create store/modules/map and add special getters to return such .env values

File: frontend/src/components/place/SearchPlace.vue [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+                mapBoxStyle: process.env.MAPBOX_STYLE,
+                filterQuery: ''
+            }
+        },
+        methods: {
+        },
+        computed: {
+            ...mapState("places", ["places"]),
+            ...mapGetters("places", ["getFilteredPlaces"]),
```
Comment (nemoipaha) [inline frontend/src/components/place/SearchPlace.vue]: Better name would be getFilteredByName.

File: frontend/src/components/place/SearchPlace.vue [hunk 7]
```diff
@@ -0,0 +1,106 @@
+<template>
+    <section class="columns">
+        <section class="column is-half">
+            <b-input class="search-field" placeholder="Find..." v-model="filterQuery"></b-input>            
+            <template v-for="(place, index) in filteredPlaces">
+                <PlaceListComponent :key="place.id" :place="place" :timer="50 * (index+1)"/>
```
Comment (nemoipaha) [inline frontend/src/components/place/SearchPlace.vue]: PlaceList accept place as param, better to have PlacePreview.vue component

File: frontend/src/store/index.js [hunk 8]
```diff
@@ -5,6 +5,7 @@ import actions from './common/actions';
 import getters from './common/getters';
 import mutations from './common/mutations';
 import auth from './modules/auth/index';
+import places from './modules/places/index';
```
Comment (nemoipaha) [inline frontend/src/store/index.js]: Module name can be just "place".

File: frontend/src/store/index.js [hunk 9]
```diff
@@ -16,5 +17,6 @@ export default new Vuex.Store({
     strict: process.env.NODE_ENV !== 'production',
     modules: {
         auth: auth,
+        places
```
Comment (nemoipaha) [inline frontend/src/store/index.js]: just place )

File: frontend/src/store/modules/places/getters.js [hunk 10]
```diff
@@ -0,0 +1,7 @@
+export default {
+    getFilteredPlaces: state => name => {
```
Comment (nemoipaha) [inline frontend/src/store/modules/places/getters.js]: Described above

File: frontend/config/dev.env.js (hunk 1 above)
Comment (aLLeXUs) [reply frontend/config/dev.env.js]: @nemoipaha variables from .env not available in frontend code, but available here

File: frontend/src/App.vue (hunk 3 above)
Comment (aLLeXUs) [reply frontend/src/App.vue]: as i know mapbox does not provide css in module https://www.mapbox.com/mapbox-gl-js/api/

File: frontend/config/dev.env.js (hunk 1 above)
Comment (nemoipaha) [reply frontend/config/dev.env.js]: @aLLeXUs it is ok, i meant why we have such `'"'` symbols in front and not just 
`MAPBOX_TOKEN: process.env.MAPBOX_TOKEN`

Comment (aLLeXUs) [reply frontend/config/dev.env.js]: @nemoipaha variable without this symbols will be object, not string, but i can move them to getter

Comment (nemoipaha) [reply frontend/config/dev.env.js]: It is ok for now )

**Pull request merged**